_articles/get-your-project-online/project-set-up-for-collaboration-with-github.md

```diff
@@ -17,9 +17,26 @@ materials: "A computer, an internet connection, and a free GitHub account"
 
 To work most effectively with contributors, your project structure should be clean and welcoming, with folder titles that make sense. For example, your project might have different folders for data, technical documentation, and images. README.md and CONTRIBUTING.md (which you’ll make in the next module) should be in your root directory, along with your ROADMAP. Think of these documents as an anchor for your project. As you add files, you may want to organize these into folders and create a consistent naming convention. Here’s an example of good project set-up.
 
+[GitHub hubot project structure](https://github.com/github/hubot)
```

Comment:
This is helpful! Thanks @nufail!

I'd still like some links to a few effectively structured github projects of various sizes / languages.